libs/components/phone-field/src/lib/modules/phone-field/phone-field.component.ts

```diff
@@ -430,11 +477,34 @@ export class SkyPhoneFieldComponent implements OnDestroy, OnInit {
         this.#adapterService.focusPhoneInput(e.element);
         this.#phoneInputAnimationTriggered = false;
       }
+
+      // Remove focus out event listeners now that country search is closed.
+      if (this.#countryFlagFocusListenerFn) {
+        this.#countryFlagFocusListenerFn();
+      }
+      if (this.#dismissCountrySearchFocusListenerFn) {
+        this.#dismissCountrySearchFocusListenerFn();
+      }
     }
 
     this.#changeDetector.markForCheck();
   }
 
+  private addFocusEventListener(el: HTMLElement): () => void {
+    return this.#renderer.listen(el, 'focusout', (event: FocusEvent) => {
+      const target = event.relatedTarget;
+      if (this.inputBoxHostSvc && target) {
+        if (!this.inputBoxHostSvc.focusIsInInput(target)) {
```

Comment:
the internal function `containsElement` was added to input box and exposed to phone field through input box host service's `focusIsInInput` function to check if focus was moved out of the input box wrapper